packages/salesforcedx-utils-vscode/test/unit/helpers/utils.test.ts

```diff
@@ -7,20 +6,99 @@
  */
 
 import { expect } from 'chai';
-import { extractJsonObject } from '../../../src/helpers';
+import * as fs from 'fs';
+import * as sinon from 'sinon';
+import {
+  extractJsonObject,
+  flushFilePath,
+  flushFilePaths
+} from '../../../src/helpers';
 
 describe('getConfigSource', () => {
-
   it('should extract a JSON string from larger string and then return as an object', async () => {
-
-    const exampleJsonString = JSON.stringify({ name: 'exampleName', error: 'exampleError' });
+    const exampleJsonString = JSON.stringify({
+      name: 'exampleName',
```

Comment:
Note:  there's a lot of format changes here in this PR.  This was not my doing.  When I save files, they are automatically being re-formatted.  I'm not sure if there was a recent configuration change, but it seems like we have a lot of PRs formatting changes lately.

Reply:
I've seen this also.  Checked in w Gordon about it yesterday.  We're going to create a ticket to look into why this is happening.